apps/reaper/lib/reaper/decoder/geojson.ex

```diff
@@ -28,7 +28,8 @@ defmodule Reaper.Decoder.GeoJson do
 
   @impl Reaper.Decoder
   def handle?(source_format) when is_binary(source_format) do
-    String.downcase(source_format) == "application/geo+json"
+    String.downcase(source_format) == "application/geo+json" ||
+      String.downcase(source_format) == "application/zip"
```

Comment:
Do we need this here? Odo should be emitting a hosted file with the source format of `application/geo+json`  if it isn't I think we should change Odo and not do this here.

Reply:
Removed this code